tsconfig.json

```diff
@@ -17,134 +17,137 @@
 		"./typings/tsd.d.ts"
 	],
 	"files": [
-		"./src/DateObject.ts",
```

Comment:
That was after a `grunt dev`.  Whatever the `updateTsconfig` task is doing in core isn't backing out those changes.